Tests/FeatherMailDriverSMTPTests/FeatherMailDriverSMTPTests.swift

```diff
@@ -51,11 +51,12 @@ final class FeatherMailDriverSMTPTests: XCTestCase {
                             password: self.pass
                         )
                     )
-                )
+                ),
+                id: MailServiceID.custom("smtp")
```

Comment:
We should use the `addMail` function with the default id, instead of explicitly providing the id & using the service `add` method.